dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/BaseClasses/BaseGeneratorSynchronous.mo

```diff
@@ -21,12 +21,12 @@ partial model BaseGeneratorSynchronous "Synchronous machine - Base dynamic model
     Placement(visible = true, transformation(origin = {0, 98}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {0, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
 
   // Input variables
-  Dynawo.Connectors.ImPin omegaRefPu(value(start = SystemBase.omegaRef0Pu)) "Reference frequency in pu";
-  Dynawo.Connectors.ImPin PmPu(value(start = Pm0Pu)) "Mechanical power in pu (base PNomTurb)";
-  Dynawo.Connectors.ImPin efdPu(value(start = Efd0Pu)) "Input voltage of exciter winding in pu (user-selected base voltage)";
+  Modelica.Blocks.Interfaces.RealInput omegaRefPu(start = SystemBase.omegaRef0Pu) "Reference frequency in pu";
+  Modelica.Blocks.Interfaces.RealInput PmPu(start = Pm0Pu) "Mechanical power in pu (base PNomTurb)";
+  Modelica.Blocks.Interfaces.RealInput efdPu(start = Efd0Pu) "Input voltage of exciter winding in pu (user-selected base voltage)";
```

Comment:
Modelica.Blocks.Interfaces.RealInput omegaRefPu(start = SystemBase.omegaRef0Pu) "Reference frequency in pu";
  Types.ActivePowerPuConnector PmPu(start = Pm0Pu) "Mechanical power in pu (base PNomTurb)";
  Types.VoltageModulePuConnector efdPu(start = Efd0Pu) "Input voltage of exciter winding in pu (user-selected base voltage)";

Reply:
Maybe a Types.AngularVelocityPuConnector should be added, so that 

Types.AngularVelocityPuConnector omegaRefPu(start = SystemBase.omegaRef0Pu) "Reference frequency in pu";

Reply:
I have implemented in the commit e24d6256019600924996f96bad71655a7e585767

Reply:
great thanks!